hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/security/token/TestZKDelegationTokenSecretManagerImpl.java

```diff
@@ -162,6 +162,7 @@ public void testMultiNodeTokenRemovalShortSyncWithoutWatch()
 
   // This is very unlikely to happen in real case, but worth putting
   // the case out
+  @SuppressWarnings("unchecked")
```

Comment:
trunk does not have suppress here.

Reply:
Yes it was added by me to fix failed javac problem reported in earlier [builds](https://github.com/apache/hadoop/pull/5067#issuecomment-1292262434). There're two similar places(line135&201) and either will fail javac. 

So two ways:
1. Keep it in sync with trunk and let javac fail for this PR. 
2.(What I did here) fix it by adding suppress warning here as I'm pretty confident those types are correct. This test file is consistent - you can see the untouched method testMultiNodeOperationWithoutWatch() has the exact same problem and has suppress warning. So I think it's applicable to the other two methods as well.

Reply:
I agree with you. Either way is fine. Will defer to @omalley for which approach to take on here.